src/lib.rs

```diff
@@ -226,24 +226,199 @@ impl WebexEventStream {
     }
 }
 
+enum Authorization<'a> {
```

Comment:
Apologies for the explicit reference marker 🙃 

Doing that here as this is mostly going to be called with `&str`.